test/ssc_test/cmod_battery_test.cpp

```diff
@@ -21,7 +21,7 @@ TEST_F(CMBattery, CommercialLifetimePeakShaving_cmod_battery) {
 		// roundtrip efficiency test will ensure that the battery cycled
 		ssc_number_t roundtripEfficiency;
 		ssc_data_get_number(data, "average_battery_roundtrip_efficiency", &roundtripEfficiency);
-		EXPECT_NEAR(roundtripEfficiency, 94.129, 2);
+		EXPECT_NEAR(roundtripEfficiency, 90.06, 2);
```

Comment:
can you explain again why this efficiency would decrease so much with the changes you've implemented?

Reply:
average_roundtrip_efficiency = 100.*(_e_discharge_accumulated / (_e_charge_accumulated + _e_loss_system));

1. the thermal model results in more heating of the battery, and adds to the losses

2. in void capacity_kibam_t::updateCapacityForThermal(double capacity_percent), thermal losses were not being added to I_loss before.

Reply:
Have you found anything on round trip efficiency in the literature? Is this closer to expected than the higher number?

Reply:
I don't think this number reflects anything real world because we don't have real world temperature coefficients. But battery efficiency is generally higher, given 20C operating conditions and not too much heating inside the battery.

As an example from https://ieeexplore.ieee.org/document/4544139 :

![image](https://user-images.githubusercontent.com/29387736/66679354-3c3eac80-ec2b-11e9-8ccc-21a26be683c7.png)